repository.json

```diff
@@ -678,6 +678,34 @@
             "version": "1.9.7",
             "url": "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.7-windows.zip",
             "fileMatchInside": ".*/phantomjs.exe$"
+        },
+        {
+            "name": "marionette",
+            "platform": "windows",
+            "bit": "32",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-v0.8.0-win32.zip"
+        },
+        {
+            "name": "marionette",
+            "platform": "linux",
+            "bit": "64",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-linux64.gz"
+        },
+        {
+            "name": "marionette",
+            "platform": "mac",
+            "bit": "32",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-OSX.gz"
+        },
+        {
+            "name": "marionette",
+            "platform": "mac",
+            "bit": "64",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-OSX.gz"
         }
```

Comment:
@lkwg82 
Yes you are right and it needs to be handled similar to how the chromedriver is handled in the `detectBits(String driverName)` method in `Repository.java`. See https://github.com/webdriverextensions/webdriverextensions-maven-plugin/blob/master/src/main/java/com/github/webdriverextensions/Repository.java#L191-L193

e.g.
```java
if (driverName.equals("marionette") && !isLinux()) {
    return "32";
}
```

Not sure how to handle the missing 32 bit linux driver though.